src/patterns/Dialog/DeleteDialog/DeleteDialog.stories.tsx

```diff
@@ -0,0 +1,66 @@
+import { Story } from '@storybook/react'
+import { useState } from 'react'
+import { ActionDialog, Stack, Text } from 'smarthr-ui'
+import styled, { css } from 'styled-components'
+
+const Wrapper = styled(Stack)`
+  ${({ theme: { spacing } }) => css`
+    padding: ${spacing.M};
```

Comment:
[余白のパターン](https://smarthr.design/products/design-patterns/spacing-layout-pattern/#h3-2) に則り 24px でお願いします!
また、16px の世界線だとTシャツサイズで表現できない余白がでてくる（12pxとか）ので、`space()` の利用を推奨したいです!

```suggestion
  ${({ theme: { space } }) => css`
    padding: ${space(1.5)};
```

Reply:
というか、DialogBody の余白決まってるなら smarthr-ui で指定しろよ感ある（やろうやろうと思ってやってないやつ

Reply:
>というか、DialogBody の余白決まってるなら smarthr-ui で指定しろよ感ある（やろうやろうと思ってやってないやつ

これは確かに〜と思いつつ、一旦`space(1.5)`で対応しました！

Reply:
:arigatau: